### PR TITLE
NEW Add methods for scrolling to top and bottom of edit form panel

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-elemental/issues/1215

### CODE REVIEW

**Review by GuySartorelli — APPROVED** [review]
LGTM, will merge when the relevant PR that uses this is ready for merging

**Comment by emteknetnz** [comment]
Have made a change after it was approved

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**
